ruby/vampires.rb

```diff
@@ -86,6 +86,7 @@
 if (name == "Drake Cula" || name == "Tu Fang" || name == "tu fang" || name == "drake cula")
```

Comment:
For actual year consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.